Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -467,6 +483,11 @@ namespace AzToolsFramework
         {
             auto entries = GetSelectedAssets();
 
+            if (entries.empty())
+            {
+                return;
+            }
+
```

Comment:
This fixes a crash I ran across while testing. If the selection model was empty, then pressing hotkeys such as delete would previously crash due to entries being empty.